go/logic/applier.go

```diff
@@ -547,6 +556,11 @@ func (this *Applier) ReadMigrationRangeValues() error {
 	}
 	defer tx.Rollback()
 
+	err = this.setOptimizerSwitch(tx)
```

Comment:
@cyrinux on 2nd thought, this applies to the whole applier connection/session and it would be less confusing to locate this call [within `.InitDBConnections()`](https://github.com/github/gh-ost/blob/eec7aa5feed3f18f65c98525a968220cb254e237/go/logic/applier.go#L84) when the connection is first opened vs in the middle

The only difference is `setOptimizerSwitch(...)` wouldn't use a transaction, the `this.db` handle _(`*sql.DB` not `*sql.Tx`)_ can be used directly instead. `.Query(...)` will work the same on the `this.db` handle